R/ssm_to_proteinpaint.R

```diff
@@ -0,0 +1,141 @@
+#' @title MAF to ProteinPaint table
+#' 
+#' @description This function takes a MAF-like data frame and convert it to the right format 
+#' for visualization with ProteinPaint.
+#' 
+#' @details For visualization with ProteinPaint, the output data frame must be saved to a 
+#' file and uploaded to https://proteinpaint.stjude.org/. 
+#' 
+#' For a valid ProteinPaint table for visualization, the MAF table must contain the required 
+#' columns to be converted to the ProteinPaint format. They are: Hugo_Symbol, RefSeq, 
+#' Chromosome, Start_Position, HGVSp_Short, Variant_Classification. Optional columns used 
+#' from the MAF table are: Mutation_Status, t_alt_count, t_depth, n_alt_count, n_depth, 
+#' Tumor_Sample_Barcode, Reference_Allele. In the case of using functions `get_ssm_by_samples`, 
+#' `get_coding_ssm`, or `get_ssm_by_patients` to get your MAF table and required/desired 
+#' columns are missing, consider using parameter `basic_columns = FALSE` to ensure that your 
+#' MAF has all needed columns. Other columns are taken from the metadata table generated 
+#' internally in `ssm_to_proteinpaint`.
+#' 
+#' @param maf A data frame in MAF format.
+#' @param print_removed_rows Boolean value indicating whether only removed rows, *i.e.* 
```

Comment:
Perhaps the wording in this parameter description can be updated to make it more clear what it does. Also, consider adding what the default value is in the parameter description. Maybe something like this would be more informative?

> Boolean parameter. Set to TRUE for returning rows from the incoming MAF that do not contain any values in the required columns. Commonly used for checking purposes. Setting this to TRUE, does not produce an output compatible with Protein paint. The default is FALSE.

Reply:
Usually this is called “verbose”

Reply:
Great improvement! Thanks!

Reply:
Usually `verbose = TRUE` means that the expected output would be printed alongside some messages. In the case of this function, the output when `print_removed_rows = TRUE` shouldn't be printed on the screen as messages, because it is too big (10620 rows if this parameter is TRUE in the example above). So the idea would be to catch this output into an object.

Reply:
Ah, thanks for explaining. From the print part of the argument name I thought it is more related to troubleshooting/sanity checking. If this is meant to be returned and stored as variable, how about calling it return_removed_rows?

Reply:
great Idea! Indeed return makes more sense then print here. Thanks!